Extension/src/LanguageServer/extension.ts

```diff
@@ -462,12 +462,22 @@ function realActivation(): void {
 
     vcpkgDbPromise = initVcpkgDatabase();
 
-    if (settings.updateChannel === 'Default') {
-        suggestInsidersChannel();
-    } else if (settings.updateChannel === 'Insiders') {
-        insiderUpdateTimer = global.setInterval(checkAndApplyUpdate, insiderUpdateTimerInterval, settings.updateChannel);
```

Comment:
Is the removal of the `insiderUpdateTimer` intentional?  I don't think it should be removed.

Reply:
Ooops, not intentional.